.goreleaser.yaml

```diff
@@ -0,0 +1,51 @@
+---
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    # Keep mod files clean
+    - go mod tidy -v
+
+builds:
+  - main: main.go
+    binary: bin/diff
+    env:
+      - CGO_ENABLED=0
+    flags: -trimpath
+    ldflags: -s -w -X {{ .ModulePath }}/cmd.Version={{ .Version }}
+    targets:
+      # helm supported targets
+      # https://github.com/helm/helm/blob/main/Makefile#L4C1-L4C1
+      - darwin_amd64
+      - darwin_arm64
+      - linux_386
+      - linux_amd64
+      - linux_arm
+      - linux_arm64
+      - linux_ppc64le
+      - linux_s390x
+      - windows_amd64
+      # additional targets
+      - freebsd_amd64
+
+archives:
+  - format: tgz
+    name_template: '{{ .ProjectName }}-{{ tolower .Os }}-{{ .Arch }}'
+    wrap_in_directory: diff
```

Comment:
I think we don't need this wrapping, because our installation script doesn't expect it.

Reply:
All files in latest release are wrapped into `diff` directory, and installation script uses it :
https://github.com/databus23/helm-diff/blob/f5cb54251336165ce792d8d7fb43329306a1a9c1/install-binary.sh#L126

Reply:
You are absolutely correct @bonddim! Thank you very much for the confirmation and the correction. You saved the day ❤️